src/core/mac/sub_mac_csl_receiver.cpp

```diff
@@ -254,6 +279,16 @@ uint32_t SubMac::GetLocalTime(void)
     return now;
 }
 
+void SubMac::LogCslWindow(uint32_t aWinStart, uint32_t aWinDuration)
+{
+#if OT_SHOULD_LOG_AT(OT_LOG_LEVEL_DEBG)
+    LogDebg("CSL window start %lu, duration %lu", ToUlong(aWinStart), ToUlong(aWinDuration));
+#else
```

Comment:
I'm confused on this. `LogDebg` will only output the log when the log level is debug. Why should we add the #if guard again?

Reply:
The `LogDebg()` is a macro. When the log level is higher than DEBUG, the body of `LogDebg()` is empty and no one uses the variable `aWinStart ` and `aWinDuration `. The compiler reports the error `-Werror=unused-parameter`. So I added the `#if` to guard the `LogDebg()`.